packages/kubernetes/1.18.13/conformance/Manifest

```diff
@@ -1,2 +0,0 @@
-image conformance k8s.gcr.io/conformance:v1.18.13
```

Comment:
was this file intended to be removed?

Reply:
yes we dont have that version for kubeadm